_includes/pbc/all/install-features/202505.0/marketplace/install-the-merchant-promotions-and-discounts-feature.md

```diff
@@ -0,0 +1,85 @@
+This document describes how to install the Merchant + Promotions & Discounts feature.
+
+## Install feature core
+
+Follow the steps below to install the feature core.
+
+### Prerequisites
+
+Install the required features:
+
+| NAME                   | VERSION          | INSTALLATION GUIDE                                                                                                                                                                                |
+|------------------------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Marketplace Merchant   | {{page.version}} | [Install the Marketplace Merchant feature](/docs/pbc/all/merchant-management/{{page.version}}/marketplace/install-and-upgrade/install-features/install-the-marketplace-merchant-feature.html)     |
```

Comment:
Should this connector be called `Marketplace Merchant + Promotions & Discounts feature`?